app/lib/email.ts

```diff
@@ -45,7 +46,11 @@ export async function sendEmail(
   try {
     await client.send(command)
   } catch (e) {
-    if (
+    if (e instanceof MessageRejected && hostname == 'localhost') {
```

Comment:
This must still throw in production.

Reply:
That should be okay though since the the hostname should be different right?

Reply:
Hmm true, think I should just let it throw normally and remove this condition?

Reply:
That's fine with me.